src/sagas/searchingGifsSaga.js

```diff
@@ -0,0 +1,18 @@
+import { call, takeEvery, put, select } from 'redux-saga/effects';
+import {searchGifs} from "../api";
+import searchingGifs from '../actions/searchingGifs';
+
+const searchingGifsSelector = state => state.searchingGifs;
+
+export function* searchGifsSaga() {
```

Comment:
Niceee, can you do something about the repeated code on both of your sagas?? How would you improve that to avoid repeting the `try catch` block for API requests???